src/commands/call.js

```diff
@@ -15,8 +15,9 @@ class CallCommand extends ApifyCommand {
         const localConfig = getLocalConfig() || {};
         const runOpts = {
             actId: args.actId || localConfig.actId,
-            waitForFinish: 120,
+            waitForFinish: 2,
         };
+        const waitForFinish = isNaN(flags.waitForFinish) ? undefined : parseInt(flags.waitForFinish, 10) * 1000;
```

Comment:
I would call this `waitForFinishMillis` so that its obvious why multiply flag by 1000.